app/domain/authentication/jwt/decoded_credentials.rb

```diff
@@ -4,23 +4,44 @@ module Jwt
     class DecodedCredentials
 
       JWT_REQUEST_BODY_FIELD_NAME = "jwt".freeze
+      # https://datatracker.ietf.org/doc/html/rfc4648#section-5
+      # jwt token is 3 blocks of base64url encoded strings separated by period '.'
+      # base64url encoding differs from regular base64 encoding as follows
+      # - padding is skipped so the pad character '=' doesn't have to be percent encoded
+      # - the 62nd and 63rd regular base64 encoding characters ('+' and '/') are replace with ('-' and '_')
+      #   The changes make the encoding alphabet file and URL safe.
+      # tokens without a signature part are denied
+      BASE_64_URL_REGEX = "[-A-Za-z0-9_=]+".freeze
+      JWT_REGEX = /\A#{BASE_64_URL_REGEX}\.#{BASE_64_URL_REGEX}\.#{BASE_64_URL_REGEX}\z/.freeze
 
       def initialize(credentials)
+        @logger = Rails.logger
```

Comment:
```suggestion
      def initialize(credentials:, logger: Rails.logger)
        @logger = logger
```

this way we can mock the logger if needed

Reply:
It's a leftover from debug automation.
I'll remove it